src/App.js

```diff
@@ -12,9 +12,50 @@ const [todoList, setTodoList] = useState([]);
 
 const [isLoading, setIsLoading] = useState(true);
 
+ const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/`;
+ console.log(url)
+ const fetchData = async() =>{
+   
+    const options = {method:'GET', headers:{'Authorization':`Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`}};
+    console.log(options)
+     
+     try {
+      const response = await
+         fetch(url, options);
+         if (!response.ok) {
+             const message = `Error: ${response.status}`;
+             throw new Error(message);
+           }
+
+           const data = await response.json();
+           
+           const todos = data.records.map((todo)=>{
+            const newTodo = {
+              id: todo.id,
+              title:todo.fields.title,
+              todo:todo.fields.todo
+            }
+
+          return newTodo;
```

Comment:
We can add an empty array in case there are not any todos that is returned from the api

```suggestion
          return newTodo || [];
```